### DISCUSSION TITLE
C.function [extern] attribute

### DISCUSSION THREAD

**Discussion by kitech** [vlang/v#11778] [Ideas]
then generate line: `extern void function();`
so no need to use extra `.h` file to declare it.

**Comment by IngwiePhoenix** [comment]
Both approaches would be great - something like this:

```v
#include <someheader.h>

fn C.myfun()

[extern]
fn C.otherfun()
```

In some cases, it might be easier to declare externals this way. Would definitively love this.

**Comment by ntrel** [comment]
`extern` would be useful to set a type for a C global variable, instead of assuming `int`.

I don't know why you would use it for a function declaration though:

```v
fn C.myfun()

[extern]
fn C.otherfun()
```
What's the difference?
